<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeveloperPage</class>
 <widget class="QMainWindow" name="DeveloperPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bugs that need fixing</string>
    </property>
   </widget>
   <widget class="QLabel" name="creation">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bug_id">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_product">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_component">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="triage_comment">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_comment">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_description">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>description</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
